CMakeLists.txt

```diff
@@ -175,7 +176,6 @@ set (COMMON_CMAKE_ARGS
   -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
   -DCMAKE_INSTALL_LIBDIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib
   -DCMAKE_PREFIX_PATH:PATH=${fletch_BUILD_INSTALL_PREFIX}
-  -DCMAKE_INSTALL_PREFIX:PATH=${fletch_BUILD_INSTALL_PREFIX}
```

Comment:
@dstoup: That was to remove a duplicate line. I separated my commits to make it more clear. Thanks for the review !